src/app/(protected)/[group]/[subGroup]/[instance]/(admin-panel)/(stage-6)/export-data/_components/send-data-section.tsx

```diff
@@ -0,0 +1,13 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+
+import { AllocationCsvData } from "@/lib/validations/allocation-csv-data";
+
+export function SendDataSection({ data }: { data: AllocationCsvData[] }) {
+  return (
+    <div>
+      <Button>Send</Button>
```

Comment:
Can we style this in the same way as the export button?

Reply:
Yep, didn't really do much on this page, just added the button there so that you can hook up your mutation to it, we can/should absolutely style it to be a bit more consistent